### ISSUE TITLE
Move up to operator-sdk 0.16.0 for s390x and ppc64le support

### ISSUE THREAD

**Issue by kaczyns** [kabanero-io/kabanero-operator#589]
operator-sdk 0.16.0 includes binaries for s390x and ppc64le.  Our multi-architecture travis builds will need this to build the operator on these platforms (it's way easier than having the build compile the sdk, or pulling it from somewhere else).

The release page lists quite a few changes so we should investigate whether we need to do anything to our code when we move up.

**Comment by dacleyra** [comment]
With the v0.16.0 sdk I see this error

```
operator-sdk generate k8s --verbose
DEBU[0000] Debug logging is set
INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [kabanero:[v1alpha1 v1alpha2], ]
F0331 14:06:24.768169    6280 deepcopy.go:885] Hit an unsupported type invalid type for invalid type, from github.com/kabanero-io/kabanero-operator/pkg/apis/kabanero/v1alpha1.Collection
```

I believe all documented & relevant migration steps are handled in the PR
https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md#v016x

Using the v0.15.1 sdk, I do not get the error.

**Comment by dacleyra** [comment]
solution: manually export GOROOT
https://github.com/operator-framework/operator-sdk/issues/2744

**Comment by kaczyns** [comment]
Fixed in #596
